app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -177,8 +178,14 @@ class UserPreference : AppCompatActivity() {
             }
 
             clearSavedPassPreference?.onPreferenceClickListener = ClickListener {
-                encryptedPreferences.edit().putString("ssh_key_local_passphrase", null).apply()
+                encryptedPreferences.edit {
+                    if (encryptedPreferences.getString("ssh_key_local_passphrase", null) != null)
+                        remove("ssh_key_local_passphrase")
+                    else if (encryptedPreferences.getString("https_password", null) != null)
+                        remove("https_password")
```

Comment:
I think we've got the orders mixed up between here and the code that sets the title.

Reply:
> I think we've got the orders mixed up between here and the code that sets the title.

I spent a solid 4 minutes here wondering did you mean. Pain is making me stupid...

Reply:
Hope you are feeling better soon!

Reply:
Just a back pain, will be gone once I'm able to get in bed. Thanks :smile: